frontend/app/src/App.tsx

```diff
@@ -1,14 +1,16 @@
-import "./App.css";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
-import HomePage from "./pages";
-import GitHubCallbackPage from "./pages/github/callback";
+import HomePage from "@/pages";
+import GitHubCallbackPage from "@/pages/github/callback";
+import { Layout } from "./components";
 
 function App() {
   return (
     <BrowserRouter>
       <Routes>
-        <Route path="/" element={<HomePage />} />
-        <Route path="/github/callback" element={<GitHubCallbackPage />} />
+        <Route path="/" element={<Layout />}>
+          <Route path="/" element={<HomePage />} />
+          <Route path="/github/callback" element={<GitHubCallbackPage />} />
```

Comment:
I think good to make constants for these routes

Reply:
completed in [5a14905](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g03/pull/8/commits/5a14905ed45bc88b6dc264136d26f455440d414e)